src/contrib/PACKAGES

```diff
@@ -3,16 +3,16 @@ Version: 0.3
 Depends: R (>= 3.0.1)
 Imports: repr, base64enc
 License: MIT + file LICENSE
-MD5sum: 23d38d54c86468ef8bebfab5518620ad
```

Comment:
Oh my, I have to fix my scripts: The script compiles (and copies) all packages and then manually commited only the new packages.

It seems that rebuilding a package will change the content (probably a date?)... I will fix this, just a second...